JapaneseFilter/sections/css_extended.txt

```diff
@@ -641,7 +641,7 @@ gadget2ch.com#?#.posts-inner:contains(/スポンサードリンク|関連コン
 libae.jp#?#.post > div.pc_display > div[id^="logly"]:upward(1)
 2ch-c.net#?#.nend_text_ad:upward(li)
 hibiki-radio.jp#?#.ng-isolate-scope > li.ng-scope ins.adsbygoogle:upward(li)
-geartics.com#?#.mb-3.text-center:contains(スポンサーリンク)
+geartics.com#?#.my-8:has(> div.ads-double > div > div > ins[data-ad-client])
```

Comment:
`#?#.my-8:has(> div.ads-double)` seems to be enough, also add `geartics.com#?#.white:has(+ .my-8 > div.ads-double) > hr:last-child`.

Reply:
I see. 
I will fix it.